README.md

```diff
@@ -122,6 +143,40 @@ Attributes:
 }
 ```
 ---
+---
+
+#### Basic Example of Module Env Variable Setup 
+
+Command: `DOCKER_API_VERSION="1.41" docker run ubuntu echo hi`
+<br> Component Attributes:
+```
+{
+    "image_name": "ubuntu",
+    "repo_digest": "sha256:04714a1bfbb2d8b5390b5cc0c055e48ebfabd4aa395821b860730ff3277ed74a",
+    "run_options": {
+        "entry_point_args": [
+            "echo",
+            "hi"
+        ]
+    }
+}
+```
+<br> Module Attributes:
+```
+"modules": [
+  {
+    "version": "0.0.5",
+    "type": "registry",
+    "name": "viam-soleng_viam-docker-manager",
+    "module_id": "viam-soleng:viam-docker-manager",
+    "env": {
```

Comment:
is this a new addition to modules?

Reply:
Yeah env variables associated with the modules were pushed to work with ROS stuff a while ago. This is just an example the Trossen has asked for so I decided to include in the readme.